### DISCUSSION TITLE
[Question]: Is there a way to debug recordings

### DISCUSSION THREAD

**Discussion by AghilasMessara** [blakeblackshear/frigate#10089] [Ask A Question]
### What is your question:

Hello everyone,
Is there a one to use a recording for debugging? I have a false negative on one motion recording. I want to use that recording and see what the detector result was on it and see how I can adjust my config.

**Comment by NickM-27** [comment]
Yes using this https://docs.frigate.video/development/contributing#2-modify-your-local-config-file-for-testing

**Comment by AghilasMessara** [comment]
Oh waw it works great!! Thank you very much.
